network/src/main/java/nl/jft/network/message/MessageHandlerChainSet.java

```diff
@@ -27,11 +27,21 @@
         Objects.requireNonNull(handler);
 
         synchronized (chains) {
-            MessageHandlerChain<M> chain = (MessageHandlerChain<M>) chains.computeIfAbsent(handler.getType(), MessageHandlerChain::new);
+            MessageHandlerChain<M> chain = getMessageHandlerChain(handler.getType());
             chain.addMessageHandler(handler);
         }
     }
 
+    private <M extends Message> MessageHandlerChain<M> getMessageHandlerChain(Class<M> type) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "getMessageHandlerChain" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUnusedPrivateMethod)